src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+
+public interface ElevatorIO {
+
+    void setPower(double power);
+
+    void getPower();
```

Comment:
I don't think you need getters in the IO, you have the inputs.

Reply:
aab075cea2b263cf440a94473ea5c8c88b2e109e

Reply:
This commit has nothing to do with the comment.

Reply:
I referenced the wrong commit, but I did fix it:)